template/tinyvue/src/views/list/search-table/index.vue

```diff
@@ -370,9 +287,15 @@
       status: '',
     },
   ) {
-    const { ...rest } = filterOptions.value;
+    let obj = {};
```

Comment:
变量名应该用有意义的名字，能够一眼看出这个变量的用途

```suggestion
    let searchInfo = {};
```

Reply:
已修改